step/src/main/java/com/google/sps/MatchManager.java

```diff
@@ -50,10 +53,18 @@ public static void generateMatch(User user) {
     }    
   }
 
-  // helper method for actually adding to people into one another's matches
+  // helper method for actually adding to people into one another's matches and sending
+  // notifications to both users
   private static void createMatch(User firstUser, User secondUser) {
     firstUser.addMatch(secondUser);
     secondUser.addMatch(firstUser);
+
+    long currTime = System.currentTimeMillis();
```

Comment:
Would it be better to have System.currentTImeMillis() be called in Notification's constructor?

Reply:
Oh that's smart, yeah.